service/models/item.py

```diff
@@ -33,9 +33,12 @@ def serialize(self):
     def deserialize(self, data):
         """Populates an Item from a dictionary"""
         try:
+            self.order_id = data["order_id"]
             self.product_name = data["product_name"]
             self.quantity = data["quantity"]
             self.price = data["price"]
+            self.created_at = data["created_at"]
```

Comment:
Oh, I realized that the "created_at" and "updated_at" attributes should also not be checked during deserialization, for the same reason as yesterday's issue. 
So it's not just here, the deserialization of order might need to be corrected as well.

Reply:
These two attributes should be automatically set by the database, not user. So the user's json should not contain them.

Reply:
Sorry for not realizing it yesterday.

Reply:
Oh god!! That makes so much sense. I'm sorry I've been a bit busy so I didn't understand exactly why that was missing. Thank you for pointing it out. I'll change the order one to as part of this PR.